deps/wazuh_testing/wazuh_testing/qa_docs/lib/config.py

```diff
@@ -32,153 +36,194 @@ def __init__(self, *args):
+        self.__read_function_regex()
+        self.__read_output_fields()
+        self.__read_test_cases_field()
+        self.__read_include_paths()
+        self.__read_include_regex()
+        self.__read_group_files()
+        self.__read_ignore_paths()
 
         if len(args) >= 3:
-            self.documentation_path = args[2]
+            self.__set_documentation_path(args[2])
         if len(args) == 4:
             # It is called with a single test to parse
             self.mode = mode.SINGLE_TEST
             self.test_name = args[3]
-            self._read_test_info()
-            self._read_module_info()
+            self.__read_test_info()
+            self.__read_module_info()
+
+
+    def __load_config_file(self, file):
+        try:
+            Config.LOGGER.debug('Loading config file')
+            with open(file) as config_file:
+                self._config_data = yaml.safe_load(config_file)
+        except:
+            raise QAValueError('Cannot load config file', Config.LOGGER.error)
+
+    def __read_test_info(self):
+        """Reads from the config file the keys to be printed from module info.
 
+        This functionality is used to print any custom field(s) you want.
+        You can use it if you only need a few fields to parse when a single test is run.
+        
+            For example you have this in your config.yaml:
 
-    def _read_test_info(self):
-        '''
-        brief: Reads from the config file the keys to be printed from test info
-        '''
+            Test info:
+                - test_wazuh_min_version: wazuh_min_version
+        """
+        Config.LOGGER.debug('Reading test info from config file')
         if 'Test info' in self._config_data:
             self.test_info = self._config_data['Test info']
+        else:
+            Config.LOGGER.warning('Cannot read test info fields')
     
-    def _read_module_info(self):
-        '''
-        brief: Reads from the config file the keys to be printed from module info
-        '''
-        if 'Module info' in self._config_data:
-            self.module_info = self._config_data['Module info']
+    def __read_module_info(self):
+        """Reads from the config file the fields to be printed from test info.
 
-    def _read_project_path(self):
-        """
-        brief: Reads from the config file the path of the project.
+        This functionality is used to print any custom field(s) you want.
+        You can use it if you only need a few fields to parse when a single test is run.
+        
+            For example you have this in your config.yaml:
+
+            Module info:
+                - test_system: os_platform
+                - test_vendor: os_vendor
+                - test_version: os_version
+                - test_target: component
         """
-        if 'Project path' in self._config_data:
-            self.project_path = self._config_data['Project path']
+        Config.LOGGER.debug('Reading module info from config file')
```

Comment:
```suggestion
        Config.LOGGER.debug('Reading module info from the config file')
```

Reply:
Done in: https://github.com/wazuh/wazuh-qa/commit/86e5045beb49f8e91dbb42b6e59f8d3164a57523